fpdf/html.py

```diff
@@ -439,6 +439,14 @@ def output_table_sep(self):
     def handle_starttag(self, tag, attrs):
         attrs = dict(attrs)
         LOGGER.debug("STARTTAG %s %s", tag, attrs)
+        # if tag == "dl":
+        #     self.pdf.ln()
+        if tag == "dt":
+            self.pdf.ln(self.h + 2)
+        if tag == "dd":
+            self.pdf.ln(self.h + 2)
+            self.indent += 2
```

Comment:
Why 2 here? I have nothing against using arbitrary values, but then please use a class constant that could be overrided by users.